src/app/pages/component-sidenav/_component-sidenav-theme.scss

```diff
@@ -1,4 +1,4 @@
-@import '~@angular/material/theming';
+@import '../../../../node_modules/@angular/material/theming';
```

Comment:
This doesn't quite appear to work still. After removing node_modules, running `npm cache clean` and doing a fresh install, I'm still seeing errors trying to resolve the `~angular/material/theming` paths.